jfix-stdlib-batching/src/main/java/ru/fix/stdlib/batching/BatchingManagerThread.java

```diff
@@ -27,7 +24,7 @@
     private final Logger log = LoggerFactory.getLogger(BatchingManagerThread.class);
 
     private final AtomicBoolean isShutdown = new AtomicBoolean();
-    private final ExecutorService batchProcessorManagerExecutor = Executors.newSingleThreadExecutor();
+    private final ExecutorService batchProcessorManagerExecutor;
     private final ThreadPoolExecutor batchProcessorPool;
```

Comment:
setThreadCount should be removed and used setMaxPoolSize from ProfiledThreadPoolExecutor instead, we fixed several bugs there, not sure whether duplicate code of setThreadCount in here do not contain bugs.

Reply:
fixed